t/t2621-job-shell-plugin-fork.t

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+test_description='Test flux-shell behavior when plugin calls fork(2)'
+
+. `dirname $0`/sharness.sh
+
+
+test_under_flux 2 job
+
+FORK_PLUGIN="${SHARNESS_TEST_DIRECTORY}/shell/plugins/.libs/fork.so"
+test_expect_success 'create shell initrc to load fork.so plugin' '
+	cat <<-EOF >fork.lua 
+	plugin.load { file = "$FORK_PLUGIN" }
+	EOF
+'
+test_expect_success 'submit sleep job with fork plugin' '
+	id=$(flux submit -o userrc=fork.lua -N2 sleep 30)
```

Comment:
i guess it doesn't matter for this particular test, since we're really testing the plugin fork, but since timeouts below are 60s, make the sleep like 100s?